RELEASES.md

```diff
@@ -1,3 +1,131 @@
+Version 1.4.0 (October 2015)
```

Comment:
We can put the date outside of the heading, that way changing the date doesn't break anchored links. Not a super big deal, it's just the anchor, but I guess it's used sometimes?